uint8 total_items
uint8 item_index
uint8 movement_type
int16 param1
int16 param2
int16 param3
